monkestation/code/datums/traits/negative.dm

```diff
@@ -63,3 +63,15 @@
 /datum/quirk/kleptomaniac/remove()
 	var/mob/living/carbon/human/H = quirk_holder
 	H.cure_trauma_type(/datum/brain_trauma/mild/kleptomania, TRAUMA_RESILIENCE_ABSOLUTE)
+
+/datum/quirk/gigantism
+	name = "Gigantism"
+	desc = "You're huge! You start the round with an immutable gigantism mutation. Even works on species without DNA!"
```

Comment:
Easily removed by taking a mutadone pill.
The only species that can't remove it would be IPCs that cannot process chems.